src/__tests__/Layout-test.js

```diff
@@ -2397,3 +2397,20 @@ describe('Layout alignContent', function() {
   testAlignContent('flex-end', 'flex-end');
   testAlignContent('flex-end', 'stretch');
 });
+
+describe('nested flex', function() {
+  it('should stretch a nested child', function() {
```

Comment:
can you use xit instead of doing a second describe? This is how I put tests that aren't passing but should